ui/js/values.js

```diff
@@ -361,6 +361,18 @@ treeherder.value("phDashboardValues",
             linkUrl: "https://wiki.mozilla.org/index.php?title=Electrolysis/Release_Criteria",
             variantDataOpt: "e10s",
             variantTitle: "e10s"
+        },
+        hasal: {
+            baseTitle: "chrome",
+            defaultRepo: "mozilla-central",
+            descP1: "Comparing performance results of Hasal.",
```

Comment:
Maybe note that the dashboard is comparing Firefox Hasal with the Chrome browser (?)